src/vivarium/framework/results/observation.py

```diff
@@ -163,7 +165,7 @@ def _wrap_results_gatherer(
 
     @staticmethod
     def create_empty_df(
-        requested_stratification_names: set[str],
+        requested_stratification_names: tuple[str, ...] | None,
```

Comment:
Why is this changed from a set to a tuple?

Reply:
I think this was originally typed incorrectly, since we actually pass in a tuple[str, ...] or None to this function.